### PR TITLE
Pull request #

### PR DESCRIPTION
**Pull request by ppxma7**
Would like to push my changes to mrTools into Julien's current branch. This shouldn't have any large-scale effects. I have made changes to the pRF plugin, but these are self-contained and should be able to operate pRF as normal. I have also added my pRF_somato plugin. 
There are a few extra arg checking loops for bugs I have run into.
Where possible, I have tried to avoid changing any legacy mrTOols functions, such as viewGet etc. Apart from one exception, where I have added an option in viewSet.m to allow group deletion without popping up a questdlg box, when passing a varargin of 'quiet'.
Thanks
Michael

### CODE REVIEW

**Comment by julienbesle** [comment]
Hi Michael,

Thanks for the contribution. 
There is a conflict in your pull request. I think it is due to a recent change in Justin's fork where he got rid of the mex version of mrDisp.m. I just merge his changes into my fork a few days ago. You can probably solve it by deleting the mrDisp executable . Can you take take care of this and update the pull request by committing your change?

I also have comments about some of the changes in fb2706e6d38568eabe324b1faf24e54272a9abb5. It looks like you have changed some defaults in the following functions:
- mrLoadRet/Edit/setFramePeriod.m
- mrLoadRet/GUI/mrPrint.m
- mrUtilities/File/FreeSurfer/mlrImportFreeSurfer.m
Unless these is a good reason for this (e.g. it fixes a bug), these defaults should not change, as they will result in unexpected behaviour for other users. If there is a good reason, then it should be explained in the commit message. Otherwise, you should make a change that does not modify the default behaviour (by adding options in the input arguments of the function or by using preferences in mrGetPref and mrSetPref).

About the changes to the original pRF code, as long as the default behaviour does not change, it should be fine, but you should add some explanations about the changes you have made (what new feature they add, what bug they fix, etc...). In fact, I think you should split this big commit into smaller independent ones and explain each of them in the corresponding commit message. To split commits, you can use git rebase -i , and then use git push --force to modify commits on your remote repository (as long as no one else is using your fork).
You should also make sure you replace any call to isfile.m by a call to mrIsFile.m.

I hope I'm not being too much of a pain. Ultimately I want to be able to push my changes to Justin's mrTools repository and this might get rejected if defaults have changed or changes are not explained (even if they leave the default behaviour unchanged).

Let me now if I can help.

Cheers
Julien

**Comment by ppxma7** [comment]
Hi Julien.
Thanks for the detailed comments.
I've reset a lot of the defaults as per your suggestions. I've avoided going down the rebasing route, as I didn't want to mess anything up by accident.
The merge should happen cleanly now. I have removed my pRF changes, and will integrate them in a future pull request.
I hope this clears everything up. Thanks
Michael

File: mrLoadRet/File/loadROI.m [hunk 1]
```diff
@@ -85,7 +85,7 @@
         s = load(pathStr{p});
         varNames = fieldnames(s);
         roi = eval(['s.',varNames{1}]);
-        roi.name = varNames{1};
+        %roi.name = varNames{1};
```
Comment (julienbesle) [inline mrLoadRet/File/loadROI.m]: I don't think this line should be commented out

File: mrUtilities/File/FreeSurfer/mlrImportFreeSurfer.m [hunk 2]
```diff
@@ -49,7 +49,7 @@
 hemiNames = {'left', 'right'};
 outDir    = fullfile(pwd,'surfRelax');
 if ~exist('volumeCropSize')
-  volumeCropSize = [176 256 256];
+  volumeCropSize = [256 256 256];
 end
```
Comment (julienbesle) [inline mrUtilities/File/FreeSurfer/mlrImportFreeSurfer.m]: This should be put back to the default value (176 in x dim)

File: mrUtilities/File/FreeSurfer/mlrImportFreeSurfer.m [hunk 3]
```diff
@@ -126,7 +126,7 @@
     end
     system(commandString);
   else
-    if mlrIsFile(outFile)
+    if isfile(outFile)
```
Comment (julienbesle) [inline mrUtilities/File/FreeSurfer/mlrImportFreeSurfer.m]: replace isfile by mlrIsFile

**Comment by julienbesle** [comment]
Hi Michael,
I just went through your two undo commits and caught a few lines that slipped through (see above). If you can just correct this, I'll merge your pull request
Cheers
Julien

**Pull request closed without merging**